packages/app/src/paths.ts

```diff
@@ -0,0 +1,5 @@
+export function getPathForPlatform (posixPath: string) {
+  if (Cypress.platform === 'win32') return posixPath.replaceAll('/', '\\')
+
+  return posixPath
+}

```

Comment:
this looks like `path.normalize()` https://nodejs.org/api/path.html#pathnormalizepath

Reply:
true, we could use that instead

Reply:
oh right, i don't think we have `path` here since it's on the client-side